code/modules/mob/living/carbon/human/species_types/vampire.dm

```diff
@@ -139,3 +143,15 @@
 	charge_max = 50
 	cooldown_min = 50
 	shapeshift_type = /mob/living/simple_animal/hostile/retaliate/bat
+/*
+/mob/living/carbon/proc/getBlood()
+	var/obj/item/organ/heart/vampire/vessel = getorgan(/obj/item/organ/heart/vampire)
+	if(!vessel)
+		return 0
+	return blood_volume
+
+/mob/living/carbon/vampire/proc/updateBloodDisplay()
+	if(hud_used) //clientless aliens
+		 
+		hud_used.vampire_blood_display.maptext = "<div align='center' valign='middle' style='position:relative; top:0px; left:6px'><font color='red'>[round(getBlood())]</font></div>"
+*/
```

Comment:
eww, commented out code.

Reply:
It's for the future, possible GUI changes